dpgen2/entrypoint/args.py

```
@@ -127,12 +130,59 @@ def dp_train_args():
     ]
 
 
+def nvnmd_train_args():
+    doc_numb_models = "Number of models trained for evaluating the model deviation"
+    doc_config = "Configuration of training"
```

Flake8 (7.2.0)

[error] local variable 'doc_config' is assigned to but never used

(F841)